site/en/install/_index.yaml

```diff
@@ -103,7 +103,7 @@ landing_page:
       path: https://js.tensorflow.org
       description: >
         TensorFlow.js is a WebGL accelerated, JavaScript library to train and
-        deploy ML models in the browser and for Node.js.
+        deploy ML models in the browser, Node.js, React native, electron.js, IOT devices and many more.
```

Comment:
I think the description on https://www.tensorflow.org/install is getting a bit long.
Can change it to "... in the browser, Node.js, and more."

Reply:
yeah, it's a little bit longer. 
what if we change it to browser, Node.js, desktop, mobile and many more

Reply:
or browser, Node.js, mobile, and more.

Reply:
f85d1a556ae3bade9d67aa1841a51c803e4c64c6